server/src/db/migrations/20230505170612_addAdminUsername.cjs

```diff
@@ -0,0 +1,22 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.alterTable("users", (table) => {
+        table.string("username").notNullable().unique()
+        table.boolean("admin").notNullable().defaultTo(false)
+    })
+}
+
+/**
+ * @param {Knex} knex
+ */
+exports.down = (knex) => {
+    return knex.schema.alterTable("users", (table) => {
+        table.dropColumn("username")
```

Comment:
the `down` action needs to be an opposite match of the `up` - you're missing dropping the admin column